lighthouse-core/test/audits/page-execution-timings-test.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const PageExecutionTimings = require('../../audits/page-execution-timings.js');
+const fs = require('fs');
+const assert = require('assert');
+
+// sadly require(file) is not working correctly.
+// traceParser parser returns preact trace data the same as JSON.parse
+// fails when require is used
+const readFile = (file, cb) => {
```

Comment:
and `readFileSync` doesn't work either!? geez what's going on with these files 😆

Reply:
Readfilesync prolly works too. I bet devtoolstimeline changes the trace

Reply:
Ah ok, `require` caches the same object so you're probably right, could we switch to readfilesync then for brevity?